plugins/lsp/default.nix

```
@@ -246,7 +246,7 @@ lib.nixvim.plugins.mkNeovimPlugin {
 
           for i, server in ipairs(__lspServers) do
             if type(server) == "string" then
-              require("lspconfig")[server].setup(__setup)
+              vim.lsp.config(server.name, __setup)
```

In this branch `server` is a string:

```suggestion
              vim.lsp.config(server, __setup)
```

This error didn't trip CI, though, so I wonder if we're missing a test-case for that?